docs/modules/ROOT/partials/setup_terraform_exoscale.adoc

```diff
@@ -0,0 +1,51 @@
+. Setup Terraform
++
+Prepare terraform
++
+[source,console]
+----
+# Set terraform image and tag to be used
+tf_image=$(\
+  yq eval ".parameters.openshift4_terraform.images.terraform.image" \
+  dependencies/openshift4-terraform/class/defaults.yml)
+tf_tag=$(\
+  yq eval ".parameters.openshift4_terraform.images.terraform.tag" \
+  dependencies/openshift4-terraform/class/defaults.yml)
+
+# Generate the terraform alias
+alias terraform='docker run -it --rm \
+  -u $(id -u) \
+  -e EXOSCALE_API_KEY \
+  -e EXOSCALE_API_SECRET \
+  -e TF_VAR_lb_exoscale_api_key \
+  -e TF_VAR_lb_exoscale_api_secret \
```

Comment:
may be worth trying to have unified instructions for Cloudscale/Exoscale by using a environment file (docker defaults to `.env` IIRC).

Reply:
docker-compose certainly uses .env by default. I'm not sure about `docker run`, the reference doesn't mention it here: https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file.

but can be made explicitly with `--env-file .env` ofc.

Reply:
Ah yeah, that's right, `docker run` doesn't have an environment file which gets picked up by default, you need to explicitly specify `--env-file`. I've got no hard feelings about the name of the file itself.

Reply:
The cloudscale instructions have snippets that reference files and dirs from the openshift4-cloudscale component. I suggest to migrate to the generic component in a separate PR resp. with the official deprecation of openshift4-cloudscale